packages/sdk/src/types/runtime/rigidbody.ts

```diff
@@ -59,6 +96,7 @@ export class RigidBody implements RigidBodyLike {
      */
 
     /**
+     * INTERNAL USE ONLY.

```

Comment:
Use @hidden instead to hide from doc generation

Reply:
Ok.  I thought that was what should be done, but I was following the example of the material.ts file like Steven had said.  I will just replace the internal use with @hidden then.